website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -85,6 +85,8 @@ In addition, one of either `identity` or `service_principal` blocks must be spec
 
 * `automatic_channel_upgrade` - (Optional) The upgrade channel for this Kubernetes Cluster. Possible values are `patch`, `rapid`, `node-image` and `stable`. Omitting this field sets this value to `none`.
 
+* `node_os_channel_upgrade` - (Optional) The upgrade channel for this Kubernetes Cluster Nodes OS Image. Possible values are `Unmanaged`, `SecurityPatch`, and `NodeImage`. Omitting this field sets this value to `none`.
+
```

Comment:
Properties in the docs are ordered alphabetically as well so this will need to be moved down. Also since it's a preview feature could we add a note on that to inform users that it must be registered first before it can be set.
```suggestion
* `node_os_channel_upgrade` - (Optional) The upgrade channel for this Kubernetes Cluster Nodes OS Image. Possible values are `Unmanaged`, `SecurityPatch`, and `NodeImage`. Omitting this field sets this value to `none`.

-> **Note:** This requires that the Preview Feature `Microsoft.ContainerService/NodeOsUpgradeChannelPreview` is enabled and the Resource Provider is re-registered, see [the documentation](https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-image#register-the-nodeosupgradechannelpreview-feature-flag) for more information.
```

Reply:
done.